### ISSUE TITLE
16.12: run as package does not work on Windows

### ISSUE THREAD

**Issue by robhagemans** [robhagemans/pcbasic#54]
Running as `python -m pcbasic` works on Linux but throws an error on Windows:
```
ERROR: Unhandled exception
Traceback (most recent call last):
  File "pcbasic\main.py", line 33, in main
  File "pcbasic\main.py", line 69, in run
  File "pcbasic\main.py", line 129, in launch_session
ImportError: cannot import name interface
```

**Comment by robhagemans** [comment]
This also breaks runner script `pcbasic.bat`

**Comment by robhagemans** [comment]
The package can be imported if we put `from . import package` at the top. I bisected the code for where the package stops being importable, and can blame the `os.chdir` at `_parse_package` in `config.py` (somewhere near line 726). 

That means we have two problems rather than one.

First problem: `python -m pcbasic` leads to a different `uargv` string than `python run.py`, which apparently leads to `pcbasic` being seen in the config as a BAZ package name (i.e. PC-BASIC package) so that this code in line 726 gets executed - it shouldn't be. This only happens in Windows because apparently Python's own `argv` is smart enough to do this correctly but we've got some homespun `argv` parsing code for Windows. IIRC the latter exists to deal correctly with unicode arguments in Python 2.7; in Python 3 we wouldn't need to do this. 

Second problem: if we *do* want to load a BAZ package, then the `os.chdir` apparently breaks later relative imports - i.e. we're unable to load packages at all this way. This happens both in Linux and in Windows - I've just checked separately and I'm unable to load a BAZ package in Linux either.

**Comment by robhagemans** [comment]
I'm going to consider this only a 16.12 problem. in 15.08 `python -m pcbasic` doesn't work at all, neither in Windows nor in Linux, because the `__import__()` of the interface modules fails. Which is a related but different issue, and is likely not worth solving in 15.08 anymore.

**Comment by robhagemans** [comment]
Fixed by commits 76bf082 and 63fb462
